### PR TITLE
Use UTF-8 encoding when reading files and ignore IntelliJ IDEA project files

### PR DESCRIPTION
**Pull request by jernejstrasner**

### CODE REVIEW

**Comment by jernejstrasner** [comment]
Huh I guess this change breaks compatibility with ree. Lemme see...

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Manifest
 pkg
 output
+.idea
```
Comment (grosser) [inline .gitignore]: if you put that in your global .gitignore, you never have to add it to any project :)
https://github.com/grosser/dotfiles/blob/master/gitconfig#L66-L67

**Comment by grosser** [comment]
encoding should not be hardcoded but taken from `Encoding.default_external`
if that is not the case then that would be a bug

File: .gitignore (hunk 1 above)
Comment (jernejstrasner) [inline .gitignore]: Yeah, but I actually do commit some files from Idea to some of my repositories so I don't keep it in the global gitignore.

**Comment by jernejstrasner** [comment]
@grosser Hmm, the interesting thing is that now that I include this gem in my project I get lots of wrong encoding errors. It seems that wherever the gem is included it switches to ASCII, while my project is heavily dependent on UTF8. I'm no Ruby expert, so I guess I need to dig into `Encoding` a little bit to see what's up.

**Comment by grosser** [comment]
It's not setting any global encoding thing, so I doubt it's just from
including it

**Comment by jernejstrasner** [comment]
Hmm, well, if I don't `require` the gem, all seems fine. Looking into it...

**Comment by jernejstrasner** [comment]
Never mind this. My environment locale settings somehow got screwed up and if I run `locale` on my machine the `LANG` env variable is empty. Huh.

**Pull request closed without merging**